### DISCUSSION TITLE
2023-08-31 Community Call

### DISCUSSION THREAD

**Discussion by gonuke** [svalinn/DAGMC#901] [Community Calls]
Present:
* UW-Madison: @gonuke, @bohmt
* ANL: @pshriwise
* NAAREA - FRANCE : @nplinden
* BaMSciConsulting : @bam241
* First Light Fusion: @shimwell
* Copenhagen Atomics: @ebknudsen
* TÜV SÜD Nuclear Technologies: @micksalt

**Comment by gonuke** [comment]
Topic: CI upgrade

**Reply by pshriwise** [reply]
- add `-Wreorder` to CI build flags. @aprilnovak found another issue related to attribute initialization ordering. Causes problems when trying to incorporate MOAB & DAGMC into MOOSE apps. (PR to fix that coming later today)
  - specification for `find_volume` (handling unmerged, coincident surfaces in `find_volume`) (https://github.com/openmc-dev/openmc/issues/2660)
  - revival of MCNP benchmarks
  - overlap check errors

**Reply by pshriwise** [reply]
Current open-source benchmark model sources:

   - https://github.com/fusion-energy/model_benchmark_zoo
   - https://nds.iaea.org/conderc/
